mobify/platform-scaffold
# Multiple coupons

Update SFCC multiple coupon support.

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/862

## Changes
- Update selectors
- Fix taxes after applying discounts

## Todos

## How to test-drive this PR
- use SFCC connector
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- checkout with $100+ items in cart
- use `MAGIC` and `AMAZE` coupons
- verify cart summarty and order summary has correct coupon and tax information
